src/trumbowyg.js

```diff
@@ -1382,6 +1382,11 @@ Object.defineProperty(jQuery.trumbowyg, 'defaultOptions', {
                         param = '<' + param + '>';
                     }
 
+                    var del = t.$ed.find('del')[0];
+
+                    if (del) {
+                        t.$ed.find('del').contents().unwrap();
+                    }
```

Comment:
I think this is not quite good solution but I can't find why execCommand doesn't work for the del button toggle on so I add this exception for at least can work.